src/main/kotlin/com/terraformation/backend/nursery/model/NurseryStats.kt

```diff
@@ -0,0 +1,37 @@
+package com.terraformation.backend.nursery.model
+
+import com.terraformation.backend.db.default_schema.FacilityId
+import com.terraformation.backend.db.nursery.WithdrawalPurpose
+
+/** Aggregated statistics for a nursery. Totals are across all batches and withdrawals. */
+data class NurseryStats(
+    val facilityId: FacilityId,
+    val totalGerminating: Long,
+    val totalNotReady: Long,
+    val totalReady: Long,
+    val totalWithdrawnByPurpose: Map<WithdrawalPurpose, Long>,
+) {
+  /**
+   * The nursery's mortality rate as a percentage. The mortality rate is calculated as the total
+   * number of plants withdrawn with a purpose of "Dead" divided by the total number of plants that
+   * have ever been in the nursery (either current inventory or already withdrawn).
+   */
+  val mortalityRate: Int
+    get() {
+      val totalDead = totalWithdrawnByPurpose[WithdrawalPurpose.Dead] ?: 0L
+
+      return if (totalDead == 0L) {
```

Comment:
Should this be arranged as
```kotlin
    val totalDead = totalWithdrawnByPurpose[WithdrawalPurpose.Dead] ?: 0L
    val totalPlants = totalInventory + totalWithdrawnByPurpose.values.sum()
    return if (totalPlants == 0L) {
      0
    } else {
      ((totalDead * 100) / totalPlants).toInt()
    }
```
to catch the divide-by-zero condition?

Reply:
If `totalDead` is nonzero, then `totalPlants` can't be zero since it includes the dead count.

Reply:
But you're right, it'd be clearer the other way. I think I was doing some premature optimization by avoiding the `sum()` call.

Reply:
Oh, I see what you mean: the `totalPlants` cannot be zero if the else block is reached and `totalDead` is non-zero.